### ISSUE TITLE
This package no longer works

### ISSUE THREAD

**Issue by CodeOtter** [Thomasdezeeuw/node-doc#8]
````
npm install -g node-doc
nodedoc dir
```

Output:
```
Converting comments to Markdown.
Input:  api/services
Output: doc
require is not defined
```

To get around this, I had to add:

```
  if(line.trim().substr(0, 7) == 'require') {
    return 'Object';
  }
```

To https://github.com/Thomasdezeeuw/node-doc/blob/v0.1.4/lib/parsers/file.js#L130

This will only work if you specify in your constants that all requires return Objects

**Comment by Thomasdezeeuw** [comment]
You're correct, I remember running into some trouble with variables including require. But I'm going to be honest I haven't worked on this package in quite some time.

I might add this fix this weekend and I'll publish a new version on npm.

**Comment by CodeOtter** [comment]
Superior.  The problem is that module, exports, and global (ironically) is locally scoped, whereas arbitrary code execution does not have access to parent scopes of its initiation expression.  Therefore, anything that uses nodeJS modules will give you issues.

**Comment by Thomasdezeeuw** [comment]
c08078f5bc115efe423a7891ce6c6b1f5a6b24cc should fix this, it's publish in v0.1.5.
